demo/api/src/app.module.ts

```diff
@@ -125,7 +125,11 @@ export class AppModule {
                 ProductsModule,
                 MenusModule,
                 DependenciesModule,
-                BrevoModule,
+                BrevoModule.register({
+                    brevo: {
```

Comment:
I'd also remove the brevo level, since it's clear that this is the config for Brevo. Will there ever be another key in the config?

Reply:
Yes there will be multiple other keys that are not related to brevo directly (eg Scope, ContentBlock, ..) that's why I kept the brevo level here.